### ISSUE TITLE
ci: set up cron workflow to pull mimic report and open pr

### ISSUE THREAD

**Issue by gosuto-inzasheru** [BalancerMaxis/protocol_fee_allocator#283]
waiting on mimic to come up with a final deadline every epoch, so we know that the report is ready

**Comment by gosuto-inzasheru** [comment]
mimic confirmed a deadline of thursdays 9am utc. should still implement some failsafe to make sure there is data, and revert in case there isnt

![Image](https://github.com/user-attachments/assets/af4383c3-056f-40c2-9d7e-74fee52d820e)

**Comment by gosuto-inzasheru** [comment]
@Tritium-VLK you said you know how to do a biweekly cron schedule? because i cant find a way

best discussion i came across is this one: https://stackoverflow.com/questions/350047/how-to-instruct-cron-to-execute-a-job-every-second-week

best solution there i could find is: `0 0 1-7,15-21 * 2`, which however fails as soon as a month has three biweekly days (eg monday or tuesday in december 2024)

i still believe the better way to go is to have a weekly cron, with logic in the python that checks if it already ran last week or not

**Comment by Tritium-VLK** [comment]
Yeah I think you're right.  Wasn't thinking a day showing up 5 times in a week.  

I can't recall the code I did it in, but thinking about it more, I just ran weekly, then took the week number and continued without doing anything if it was even.   Let me know if you want me to find a code example.
